src/Service/Storage/S3.php

```diff
@@ -317,8 +334,15 @@ protected function addInputOptions(Application $app)
         $definition->addOption(new InputOption(
             "bucket",
             null,
-            InputOption::VALUE_REQUIRED,
-            "S3 bucket"
+            InputOption::VALUE_OPTIONAL,
```

Comment:
Do we want people to be able to specify the `--bucket` option without giving it a value? Or am I misunderstanding how `VALUE_OPTIONAL` works?

Reply:
Good spot. We might be dealing with a data export so we don't care about `--bucket`, we care about `--data-bucket`. The validation for this is in `validateConfiguration()` a little further down.